understanding-jwt/README.md

```diff
@@ -0,0 +1,206 @@
+## What is in store?
+Who does not like a story while learning a technical concept? Let us try to get a grasp of what we would learn via a story. Imagine going to a bank to open an account. You submit all documents, get all verification done and then once the account is ready, you get a passbook. The next time you go to the bank for any purpose like cash-withdrawal or cheque deposition, you do not need to do a full document verification again. As long as you have the passbook to identify yourself, you are good to go. Now you might be wondering, how is this related to security? Ever thought how you sign-in to an app and you stay signed and you do not have to enter the password for every page you navigate to? One of the secure ways to do this is using JWT. Are you excited to learn? Follow along! 
+
+## Introduction: What, Where and Why?
+
+JWT is a way for securely transmitting information across the network as a JSON object. This information can be verified and trusted because it is digitally signed.
+
+> Well, what does JWT stand for? 
+
+JWT is the abbreviation for JSON Web Token. 
+
+> Now what is JSON? 
+
+That is secret. Actually no. In the spirit of 'Learn by Doing', it will be awesome if you do a bit of research on it if you are hearing it for the first time.
+
+> What does it look like?
+
+`eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJDcmlvLmRvIGhlbHBzIHlvdSBsZWFybiBieSBkb2luZyEifQ.QKVJvVR80IMeoRzcLPWh6AIceQs-HkNlMey4swkHOX0`
+
+Intimidating? Let us try to break it down. If you look closely, it contains three parts separated by two `.`.
+
+```
+eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.
+eyJzdWIiOiJDcmlvLmRvIGhlbHBzIHlvdSBsZWFybiBieSBkb2luZyEifQ.
+QKVJvVR80IMeoRzcLPWh6AIceQs-HkNlMey4swkHOX0
+```
+Now it is a bit more precise.
+
+> But where is the JSON? This looks like random characters and not JSON.
+
+Yes, you are right; this is not a standard JSON at first sight. This is base64url encoded JSON. base64url is a way to convert a human-readable string into URL friendly string. So, what you saw above is a JSON itself, just a different representation.
+
+The three parts of JWT are
+- ** Header**: The Header is used to give the information to the parser about the token itself. The Header typically consists of the type of the token, which is JWT, and the signing algorithm being used. 
+- ** Payload**: The second part of the token is the Payload, which contains the actual information 
+- ** Signature**: Signature is used to verify the token was not changed along the way. This prevents tampering.
```

Comment:
It seems like you were trying to bold the above three words. It's not working because of the extra space you have after the ** at the beginning.

Reply:
Fixed in updated commits